node-graph/graph-craft/src/graphene_compiler.rs

```diff
@@ -8,7 +8,7 @@ use crate::proto::{LocalFuture, ProtoNetwork};
 pub struct Compiler {}
 
 impl Compiler {
-	pub fn compile(&self, mut network: NodeNetwork) -> impl Iterator<Item = ProtoNetwork> {
```

Comment:
I think it would make more sense to return `Result<Iterator>` instead

Reply:
Thanks, will make that change!